url_shortener/shorten_app/migrations/0001_initial.py

```diff
@@ -0,0 +1,22 @@
+# Generated by Django 3.1.1 on 2021-01-26 17:29
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    initial = True
+
+    dependencies = [
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='UrlData',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('url', models.CharField(max_length=200)),
+                ('short', models.CharField(max_length=15)),
+            ],
+        ),
+    ]
```

Comment:
db setup looks good

question - should there be a max length for the long URL? Isn't that why users are using the app, to shorten their crazy long URL.